src/components/InlineErrorText.js

```diff
@@ -7,10 +7,14 @@ import Text from './Text';
 const propTypes = {
     /** Text to display */
     children: PropTypes.string,
+
+    /** Styling for inline error text */
+    additionalStyles: PropTypes.oneOfType(PropTypes.array),
```

Comment:
Styles should be of type `PropTypes.arrayOf(PropTypes.object)`
Also, just `styles` instead of `additionalStyles` is fine.